target/linux/octeon/image/Makefile

```diff
@@ -94,4 +94,23 @@ define Device/ubnt_unifi-usg
 endef
 TARGET_DEVICES += ubnt_unifi-usg
 
+define Device/n821
```

Comment:
The device name should follow the proper compatible / dts name, see the other definitions.